### PR TITLE
Make rfc format

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: rfc_frontmatter.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+% Title = "FF Video Codec 1"
+% abbrev = "FFV1"
+% category = "info"
```
Comment (retokromer) [inline rfc_frontmatter.md]: Should be:
`category = "std"`

File: rfc_frontmatter.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+% docName = ""
```
Comment (retokromer) [inline rfc_frontmatter.md]: You should give some input, e.g.:
`docName = "draft-niedermayer-cellar-ebml-00"`

**Comment by dericed** [comment]
I reviewed the pr to incorporate @retokromer's comments. I also added new commits in order to get an output that is acceptable to rfc checkers. Then force pushed the result. Please review.

**Comment by michaelni** [comment]
This pull request is a mess, if its merged multiple commits appear duplicated like
" Reformatting before SliceContent changes" or "SliceContent description update"

**Comment by dericed** [comment]
@michaelni I did some cleanup and force pushed. Please re-review.

**Pull request merged**